src/users/components/UserList.vue

```diff
@@ -92,6 +117,13 @@ export default {
       if (!this.filterTerm || this.filterTerm === '') return list
       return list.filter(u => u.displayName.toLowerCase().includes(this.filterTerm.toLowerCase()))
     },
+    inactivityInfo () {
+      Dialog.create({
+        title: this.$t('INACTIVITY.WHAT'),
+        message: this.$t('INACTIVITY.HELP'),
```

Comment:
Need to add `dayCount` as argument.

Actually, we should get the value from backend to keep settings in sync. We could add it to the group detail endpoint. It's probably a task for me then...

Reply:
Yep. Tell me when it's there, then I'll change it. :)

Reply:
It's there! `group.memberInactiveAfterDays`